core/storage/base_model/gae_models.py

```diff
@@ -409,14 +530,23 @@ def get(cls, entity_id, strict=True, version=None):
     @classmethod
     def get_snapshots_metadata(
             cls, model_instance_id, version_numbers, allow_deleted=False):
-        """Returns a list of dicts, each representing a model snapshot.
+        """Gets a list of dicts, each representing a model snapshot.
 
         One dict is returned for each version number in the list of version
         numbers requested. If any of the version numbers does not exist, an
         error is raised.
 
-        If `allow_deleted` is False, an error is raised if the current model
-        has been deleted.
+        Args:
+            model_instance_id: Id of requested model.
+            version_numbers: List of version numbers.
+            allow_deleted: bool. If is False, an error is raised if the current
+                model has been deleted. Default is False.
+
+        Returns:
+            List of dicts, each dict representing a model snapshot.
+
+        Raises:
+            Exception: If any of version numbers does not exist.
```

Comment:
Exception: There is no model instance corresponding to at least one of the given version numbers.